base/data/nlu/more.md

```diff
@@ -12,3 +12,7 @@
 - ℹ️
 - Moee
 - m0re
+- okwengeziwe
+- okuninzi
+- tse ding tse ngata
```

Comment:
Hmm, this doesn't seem like a great keyword, I'm not sure how well the NLU will deal with misspellings, or only sending parts of the keyword. I guess we'll only know once we get real-world data.

Reply:
👍